<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetAction">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Magasin</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Libelle :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditSearchMagasinLibelle"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableViewMagasin">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonMagasinAjouter">
            <property name="text">
             <string>Ajouter</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/add</normaloff>:/add</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonMagasinModifier">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Modifier</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/edit</normaloff>:/edit</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonMagasinOrganiser">
            <property name="text">
             <string>Organiser</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/organize</normaloff>:/organize</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonMagasinSupprimer">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Supprimer</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/del</normaloff>:/del</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Rayon</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Libelle :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="lineEditSearchRayonLibelle"/>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableView" name="tableViewRayon">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonRayonAjouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/add</normaloff>:/add</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButtonRayonModifier">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/edit</normaloff>:/edit</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButtonRayonSupprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/del</normaloff>:/del</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Produit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Libelle :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="lineEditSearchProduitLibelle"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Rayon :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="ComboBoxRayon" name="comboBoxSearchProduitRayon"/>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QTableView" name="tableViewProduit">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButtonProduitAjouter">
          <property name="text">
           <string>Ajouter</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/add</normaloff>:/add</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButtonProduitModifier">
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/edit</normaloff>:/edit</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButtonProduitSupprimer">
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/del</normaloff>:/del</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionMagasin"/>
    <addaction name="actionRayon"/>
    <addaction name="actionProduit"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionMagasin">
   <property name="text">
    <string>Magasin</string>
   </property>
  </action>
  <action name="actionRayon">
   <property name="text">
    <string>Rayon</string>
   </property>
  </action>
  <action name="actionProduit">
   <property name="text">
    <string>Produit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ComboBoxRayon</class>
   <extends>QComboBox</extends>
   <header>comboboxrayon.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
